src/tailwind.src.css

```diff
@@ -2,4 +2,7 @@
 
 @tailwind preflight;
 @tailwind components;
+
+@import './components/Button/Button.css';
```

Comment:
🔥 not needed since the button plugin pulls the styles into compiled stylesheet, you also need to use https://github.com/TrySound/postcss-easy-import to import css into the main stylesheet otherwise it just copies this directly into the compiled styles as is